app/helpers/catalog_helper.rb

```diff
@@ -123,4 +123,16 @@ def articles_plus_url
   def my_library_card_url
     'https://emory.primo.exlibrisgroup.com/discovery/account?vid=01GALI_EMORY:services&section=overview&lang=en'
   end
+
+  def rounded_lightbulb
+    tag.div(
+      tag.span(image_tag("lightbulb.svg", height: "32", width: "32"), class: "rounded-lightbulb"),
+      class: "lightbulb"
+    )
+  end
+
+  def availability_present?(doc_avail_values, document)
```

Comment:
Yeah, I left this there because I knew you were changing code related to availability and requesting. I knew that it would dry up the repetitive testing on two different files. Just pushed up 8505e28 that apples the helper to your code.